couple changes

cheeze2000 opened the pull request.

nicholastanvis committed:
couple changes

Valen-H committed:
Update avatar.js

Valen-H committed:
Optimisations

By Cheeze

ValentinHacker commented:
All command-file contents should follow similar patterns. All command main functions are async because they are all called with await by default and must be unref'd from main process (for speed).

Valen-H committed:
Oprimisation

By Cheeze

Valen-H committed:
Rejected

I prefer safe regexes.

Valen-H committed:
Optimisation

By Cheeze

Valen-H committed:
Optimisation

By Cheeze

Valen-H committed:
Rejected

Awaits are used to throttle errors to try-catch block rather than promise-catch

Valen-H committed:
Optimisation

By Cheeze

Valen-H reviewed (approved):
Ready.
